samples/DataGridListSample.js

```diff
@@ -87,8 +87,8 @@ enyo.kind({
 enyo.kind({
 	name: "moon.sample.GridSampleItem",
 	kind: "moon.GridListImageItem",
-	mixins: ["moon.SelectionOverlaySupport"],
-	selectionOverlayVerticalOffset: 35,
```

Comment:
I think we still need sample code for ```moon.SelectionOverlaySupport```
How about creating ```moon.sample.GridSampleVideoItem``` inherited from ```moon.BadgeOverlaySupport```
And you can add ToggleButton in DataGridList sample to change between ```moon.SelectionOverlaySupport``` and ```moon.BadgeOverlaySupport```